scripts/clib/readme.txt

```diff
@@ -0,0 +1,11 @@
+To use, write down your local cpython path in clib_path.txt, and run,
```

Comment:
could better yet be an `.md` for auto rendering by github.

Reply:
:+1: